src/lammpsparser/potential.py

```
@@ -33,6 +33,383 @@
 
 """
 
+import re
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Set, Tuple, Union
```

Ruff (0.15.0)

[error] Redefinition of unused `Optional` from line 3: `Optional` redefined here

Remove definition: `Optional`

(F811)